casper/src/test/scala/coop/rchain/casper/util/rholang/RuntimeManagerTest.scala

```diff
@@ -71,7 +71,7 @@ class RuntimeManagerTest extends FlatSpec with Matchers {
     result.status.isFailed should be(true)
   }
 
-  "computeDeployPayment" should "transfer REV from the deployer's vault to the PoS vault" in withTestFixture {
+  "computeDeployPayment" should "transfer REV from the deployer's vault to the PoS vault" ignore withTestFixture {
```

Comment:
Please explain why the changes to runtime manager are needed.

Reply:
There shouldn't be any changes to runtime manager. It's working against the `PoS.rho` interface, which hasn't changed from what it was before you took over.